gustaf/io/mfem.py

```diff
@@ -198,6 +202,58 @@ def format_array(array):
 
     # Export 3D mesh
     else:
-        raise NotImplementedError(
-            f"Sorry, we cannot export mesh of dimension {dim}."
+        # Elements
+        element_attribute = 1  # Other numbers not yet supported
+        elements = mesh.elements
+        n_elements, n_element_vertices = elements.shape
+        if n_element_vertices == 4:
+            geometry_type = geometry_types["TETRAHEDRON"]
+        else:
+            raise NotImplementedError(
+                "Sorry, we cannot export mesh with elements "
+                f"with {n_element_vertices} vertices."
+            )
+        e = np.ones((n_elements, 1), dtype=settings.INT_DTYPE)
+        elements_array = np.hstack(
+            (element_attribute * e, geometry_type * e, elements)
         )
+        elements_array_string = format_array(elements_array)
+        elements_string = f"elements\n{n_elements}\n"
+        elements_string += f"{elements_array_string}\n\n"
+
+        # Boundary
+        faces = mesh.faces()
+        if BC is None:
+            BC = {1: mesh.to_faces(False).single_faces()}
```

Comment:
This would not be helpful.

Reply:
It would just add an additional variable that is only used for this. Also, it would add some logic somewhere down the line (next few lines to overwrite the BC variable anyway, so it would just add a variable and complexity. What could be done is to add a comment: "No boundary conditions provided; generating one for all faces."